posix.mak

```diff
@@ -522,6 +522,9 @@ style: ../dscanner/dsc
 	@echo "Enforce space between a .. b"
 	grep -nrE '[[:alnum:]][.][.][[:alnum:]]|[[:alnum:]] [.][.][[:alnum:]]|[[:alnum:]][.][.] [[:alnum:]]' $$(find . -name '*.d' | grep -vE 'std/string.d|std/uni.d') ; test $$? -eq 1
 
+	@echo "Check for unnecessary parentheses after return (doesn't catch everything)"
```

Comment:
Actually this checks for space after `return`. BTW it should also check for things like `return-1;` or `return!condition;` etc.